services/dynamic-sidecar/tests/unit/test_modules_notifier.py

```diff
@@ -217,7 +217,7 @@ async def test_notifier_publish_disk_usage(
                 jsonable_encoder(ServiceDiskUsage(node_id=node_id, usage=usage))
             )
 
-    await _assert_call_count(server_disconnect, call_count=_NUMBER_OF_CLIENTS)
+    await _assert_call_count(server_disconnect, call_count=_NUMBER_OF_CLIENTS * 2)
```

Comment:
i am curious: why the number of calls are now multiplied by two due to a library update? At first sight it is  very suspicious that the function is called twice more than in the previous versions

Reply:
After the library upgrade you now get two events from socket.io one of them contains the socket_id the other says something like "disconnected"